store/copr/batch_coprocessor.go

```diff
@@ -798,15 +781,108 @@ func failpointCheckWhichPolicy(act tiflashcompute.DispatchPolicy) {
 	})
 }
 
+func filterAllStoresAccordingToTiFlashReplicaRead(allStores []uint64, aliveStoreIDs map[uint64]struct{}, aliveStoreIDsInTiDBZone map[uint64]struct{}, isTiDBZoneSet bool, policy tiflash.ReplicaRead) (storesMatchedPolicy []uint64, needsCrossZoneAccess bool) {
```

Comment:
If we can overwrite `policy` to `allreplicas` when `isTiDBZoneSet` is false, we don't need pass `isTiDBZoneSet` as an argument in this function.